src/workflows/workflows.ts

```diff
@@ -64,17 +65,17 @@ export class Workflows {
   private readonly usersApi: UsersApi;
   private readonly withdrawalsApi: WithdrawalsApi;
 
-  constructor(protected config: Config) {
+  constructor(protected config: Config, basePath?: string, axiosInstance?: AxiosInstance) {
```

Comment:
Looks fine to me. 
Just one thing to note is for the code to actually use this basePath, config.api.basePath must be falsy.